frontend/src/services/events.js

```diff
@@ -1,6 +1,8 @@
 import axios from 'axios'
 
-export const getEvents = async () => {
-    const response = await axios.get('/api/events')
+export const getEvents = async (language, resultLimit, tags) => {
+    const response = await axios.get(
+        'api/events' + '/' + language + '/' + resultLimit + '/' + tags,
+    )
```

Comment:
tää on ok, oisko eslintin asetuksissa että string conc. pitäis hoitaa `api/events/${language}` etc